ui/modal/create_password_modal.go

```diff
@@ -184,7 +208,25 @@ func (cm *CreatePasswordModal) Layout(gtx layout.Context) D {
 			return layout.Dimensions{}
 		},
 		func(gtx C) D {
-			return cm.passwordEditor.Layout(gtx)
+			return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+				layout.Rigid(cm.passwordEditor.Layout),
+				layout.Rigid(func(gtx C) D {
+					return layout.Inset{Left: values.MarginPadding20, Right: values.MarginPadding20}.Layout(gtx, func(gtx C) D {
+						return layout.Flex{Spacing: layout.SpaceBetween}.Layout(gtx,
+							layout.Rigid(func(gtx C) D {
+								txt := cm.Theme.Label(values.MarginPadding14, "This spending password is for the new wallet only")
```

Comment:
Why was this text added here? It's not included in the mockup and it shows up if I try to setup a startup password in settings.

Reply:
<img width="389" alt="image" src="https://user-images.githubusercontent.com/27733432/133384420-cb25a17b-3115-4d02-8bda-3424a595930a.png">
it is in the mock up..
i will fix the setting page.